src/views/SeedPhrase/ConfirmSeed.vue

```diff
@@ -0,0 +1,127 @@
+<template>
+
+    <div class="login-wrapper confirm-seed h-100 no-outer-pad">
+        <div class="confirm-seed_logo-wrap mt-5 mb-3 mx-auto">
+            <img :src="logo"/>
+        </div>
+        <div class="confirm-seed_top">
+            <h2>Confirm Seed Phrase</h2>
+            <p class="px-3">Tap the 3 words matching their position in the seed phrase.  Once confirmed, store the phrase securely.</p>
+        </div>
+        <div class="confirm-seed_bottom">
+            <div class="confirm-seed_bottom-top">
+                <div class="mx-auto mt-2">
+                <form class="form">
+                    <div class="confirm-seed_inputs d-flex flex-row justify-content-around">
+                        <div>
+                            <label>1<sup>st</sup> Word</label>
+                            <input type="text" class="form-control text-center w-75 mx-auto form-control-sm" v-model="phraseSnip[0]" autocomplete="off" required />
+                        </div>
+                        <div>
+                            <label>5<sup>th</sup> Word</label>
+                            <input type="text" class="form-control text-center w-75 mx-auto form-control-sm" v-model="phraseSnip[1]" autocomplete="off" required />
+                        </div>
+                        <div>
+                            <label>12<sup>th</sup> Word</label>
+                            <input type="text" class="form-control text-center w-75 mx-auto form-control-sm" v-model="phraseSnip[2]" autocomplete="off" required />
+                        </div>
+                    </div>
+                </form>
+                </div>
+                <div class="confirm-seed_selections mt-3 px-1 mx-auto">
+                    <div class="confirm-seed_options flex-container d-flex flex-wrap justify-content-around px-1">
+                        <button class="btn button-confirm px-1 py-0 mx-1 my-3" v-for="(word, i) in seedList" :disabled="phraseIndex[i]" :key="word" @click="onSelect(word, i)">{{ word }}</button>
+                    </div>
+                </div>
+            </div>
+        </div>
+        <div class="footer-container bg-white px-2">
+            <div class="button-group">
+                <router-link to="/backup"><button class="btn btn-outline-primary btn-lg btn-block">Back</button></router-link>
+                <router-link to="/congratulations"><button class="btn btn-primary btn-lg btn-block" :disabled="!isConfirmedMatch">Continue</button></router-link>
```

Comment:
I think you could bypass the confirmation by closing your wallet and reopening it here. 
Is there something we can do to prevent that? Like detecting that and deleting the wallet so that next time it starts fresh? 
The problem is, the wallet is committed to the state before this.